src/stories/Button/component.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { action } from "@storybook/addon-actions";
+import StyledButton from "./index";
+
+const defaultText = "Default";
```

Comment:
Your StyledButton component will take props that would include, 1) text & 2) function for onClick

example:

const props = {
    buttonText: 'Text one',
   onClick= () => null
}

Reply:
done